internal/commands/sbomtest/sbomtest.go

```diff
@@ -76,6 +78,10 @@ func TestWorkflow(
 
 	err = sbomTest.WaitUntilComplete(ctx, errFactory)
 	if err != nil {
+		var snykErr snyk_errors.Error
+		if stderrors.As(err, &snykErr) {
+			return nil, snykErr
```

Comment:
Isn't that just a new interface pointing to the same underlaying instance, probably just me being confused, but I think it's the same thing as returning err.